db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_remove_vm_finished.yaml

```diff
@@ -9,8 +9,8 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/src_vm_disconnect_storage"
-  - rel5:
       value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_unregister&param="
+  - rel5:
+      value: "/System/event_handlers/event_action_refresh_sync?target=src_vm"
```

Comment:
Depends if RHEV can do that? Maybe only save it from Storage and link it from Hardware? But even saving it from 2 places should work, we just need to use the same unique index and then the saving code should create it in one place and update it in another. (we just need to make sure it has the same data)